### ISSUE TITLE
Create load method for SimpleModel classes

### ISSUE THREAD

**Issue by skekre98** [skekre98/simple_learn#56]
**Description of feature/enhancement**
Create a method to load a SimpleModel archive zip file

**Description of implementation**
Would recommend using the built in python [zipfile](https://pymotw.com/2/zipfile/) library

**Why this is a good feature/enhancement**
Would allow for model persistence

**Additional context**
n/a

**Comment by Turtle24** [comment]
@skekre98 is this kind of what you're looking for?

```python
    def load_model(self, file_path):
        zf = zipfile.ZipFile(file_path)
        for filename in [ 'attr.txt']:
            try:
                data = zf.read(filename)
                data = data.decode('utf8').replace("'", '"')
            except KeyError:
                print('ERROR: Did not find {filename} in zip file').format(filename)
            else:
                print(data)
                
```
***Output:***
```
>>> import simple_learn
>>> from simple_learn.classifiers import SimpleClassifier
>>> clf = SimpleClassifier()  
>>> clf.load_model('test.zip')
{
    "Type": "KNeighborsClassifier",
    "Training Duration": "0.0010001659393310547s",
    "GridSearch Duration": "0.09302067756652832s",
    "Parameters": {
        "metric": "euclidean",
        "n_neighbors": 4,
        "weights": "uniform"
    },
    "Metrics": {
        "Training Accuracy": 0.9866666666666667,  
        "Jaccard Score": 0.9245283018867925,      
        "F1 Score": 0.96
    }
}
```

**Comment by skekre98** [comment]
Yes, I am currently able to load the json file into the model, but I am trying to figure out a workaround to loading the joblib file. `zf.read` unfortunately returns the byte data of the file and not the filepath which is required for `joblib.load`.

**Comment by Turtle24** [comment]
> Yes, I am currently able to load the json file into the model, but I am trying to figure out a workaround to loading the joblib file. `zf.read` unfortunately returns the byte data of the file and not the filepath which is required for `joblib.load`.

Yeah I had the same issue with bytes when I tried to return a json.loads and the ```data.decode('utf8').replace("'", '"')``` fixed it. Hope that helps.

**Comment by skekre98** [comment]
I was able to get the json file loaded, but I'm currently working on getting the joblib file loaded for the sklearn model. I think `zf.extract` is what we are looking for.

**Comment by skekre98** [comment]
@Turtle24 I've added the fixes/updates in #57 if you wanted to take a look at it.
